app/Models/Topic.php

```diff
@@ -63,4 +64,15 @@ public function follows()
     {
         return $this->morphMany('App\Models\Follow', 'follow');
     }
+
+//    follow topic theo user
+    public function followed($topicId){
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Models/Topic.php</br>**Line**: 69</br>Opening brace should be on a new line